src/pages/library/DataBrowser.js

```diff
@@ -16,6 +18,8 @@ import { importDataToWorkspace, snapshotAccessTypes, useDataCatalog } from 'src/
 import { RequestDatasetAccessModal } from 'src/pages/library/RequestDatasetAccessModal'
 
 
+export const acknowledgmentStore = staticStorageSlot(localStorage, `catalog-beta-acknowledgment-${getUser().id}`)
```

Comment:
Apologies for the churn on this - when testing, it looks like the userId isn't populated when this code is initially run.
I have a few changes that should correct the issue and allow us to capture the response for a specific user:
```suggestion
export const acknowledgmentStore = staticStorageSlot(localStorage, 'catalog-beta-acknowledgment')
```